R/mobr_boxplots.R

```diff
@@ -64,11 +66,16 @@ calc_biodiv_single_group <- function(abund_vec, n_rare){
    if (length(abund_vec) > 0){
       out_vec["N"] <- sum(abund_vec)
       out_vec["S"] <- length(abund_vec)
-      if (!is.na(n_rare))
-         out_vec["S_rare"] <- rarefaction(abund_vec, method = "indiv", effort = n_rare)
-      out_vec["PIE"] <- calc_PIE(abund_vec)
-      out_vec["ENS_PIE"] <- 1/(1 - out_vec["PIE"])
       
+      inext1 <- iNEXT::iNEXT(abund_vec, q = 0, datatype = "abundance", size = n_rare, se = F)
+      out_vec["S_rare1"] <- inext1$iNextEst$qD[1] 
```

Comment:
the reason `iNEXT` is used is because plots are sampled with replacement and therefore the bootstrapped N may be larger than the observed N and therefore the rarefied S needs to be extrapolated. This package uses a method developed by Colwell to carry out the extrapolation. I suggested another solution may be to draw from the metacommunity SAD as generated by the Jade package for our significant tests in the continuous scale analysis (see #54 and #143). Let's discuss as a group what is the best option here. Ideally we will have some continuity between the method used to derive extrapolated S, extrapolated SAD, and the extrapolated rarefaction curve.